### DISCUSSION TITLE
Change menu bar icon for better visibility

### DISCUSSION THREAD

**Discussion by kevintoepfer** [MonitorControl/MonitorControl#1284] [General]
### Before opening the issue, have you...?

- [x] Searched for existing issues

### Is your feature request related to a problem? Please describe

Using an icon from SF Symbols makes it difficult to distinguish MonitorControl from other apps. Also, not all users have brightness adjustment as their primary purpose. `sun.max` is not an optimal icon for these users.

Issues #766, #748 and #632 all lead to an (silent) option for choosing a custom icon.

### Describe the solution you'd like

I suggest to use an own icon instead of an icon from SF Symbols. The icon should stay in line with Apples [Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/macos/extensions/menu-bar-extras/).

#### Before

[Fluor](https://github.com/Pyroh/Fluor) as an example right next to MonitorControl.

![before](https://user-images.githubusercontent.com/53897272/151978212-ba6b37f8-9f9a-4ed6-a5b3-064c4cede696.png)

#### After (Layout)

**Variant 1**
(modified `sun.max`from SF Symbols)

![after-01](https://user-images.githubusercontent.com/53897272/151978412-7b35ac28-e96f-4be6-a483-1f94b76d4892.png)

**Variant 2**
(modified `display` or `lock.display` from SF Symbols)

![after-02](https://user-images.githubusercontent.com/53897272/151978485-3c44b9cb-148f-4a1d-b935-e110e733bbd1.png)

### Describe alternatives you've considered

See [Custom icons by editing the preferences plist file #766](https://github.com/MonitorControl/MonitorControl/issues/766).

### Anything else?

I will be happy to assist with additional variations or the exported icon file if desired.

**Comment by waydabber** [comment]
There was a similar pull request lately - https://github.com/MonitorControl/MonitorControl/pull/880

The icon can be changed but needs the unanimous consent of all devs. I am flexible on the issue. There is a request on a monthly basis about changing the menu icon so I understand that at least some percentage of the community is not happy with the current one. :)

**Comment by waydabber** [comment]
@the0neyouseek @JoniVR
